editor/editor_file_system.cpp

```diff
@@ -674,6 +675,29 @@ bool EditorFileSystem::_scan_import_support(const Vector<String> &reimports) {
 bool EditorFileSystem::_update_scan_actions() {
 	sources_changed.clear();
 
+	if (first_scan) {
```

Comment:
I'm concern about loading the theme here. Technically, if an addon loads a scene with controls, these controls would be already loaded by now because they are loaded in `_first_scan_filesystem`. I'm not that familiar with themes and gui nodes but I guess that these nodes would not have the correct theme on loading. It's probably not a big problem because when the theme changes the controls should update.

I'm not sure it's the best idea but maybe the code that was added here should be put in the method like `init_project_theme` and called in `_first_scan_filesystem`, just before calling `init_autoloads`?